www/src/components/CodeBlock.jsx

```diff
@@ -57,9 +56,9 @@ function CodeBlock({ children, className, live }) {
             useEffect,
             useState,
             useMemo,
-            FontAwesome,
             MiyazakiCard,
             HipsterIpsum,
+            MenuIcon: ParagonIcons.Menu,
```

Comment:
There is already a `Menu` component in the scope from Paragon, so ween to provide Menu icon under other name to use it.

Reply:
Makes sense!